### PR TITLE
Add site skeleton

### LINKED ISSUE #224: Add site skeleton

### PR DESCRIPTION
**Pull request by ddeboer**
Fix #224.

To do:
- [ ] document Zotonic addsite command to create a site based on the Ginger skeleton

### CODE REVIEW

**Comment by DorienD** [comment]
Is it then also possible to create a Make command for this in docker?

**Comment by ddeboer** [comment]
Yes/

File: skel/gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    ];
+
+    gulp.watch(watchPaths, ['sass']);
+});
+
+function handleError(e) {
+    console.log('Error: ', e);
+}
+
+gulp.task('default', ['sass', 'sass:watch']);
```
Comment (ddeboer) [inline skel/gulpfile.js]: @Devenwel Does this gulpfile look right to you?

Comment (Devenwel) [inline skel/gulpfile.js]: Seems to be the most up-to-date version, yes.

**Pull request merged**